focus-areas/issue-resolution/issues-active.md

```diff
@@ -1,65 +1,20 @@
-# Issues Active
+# **Issues Active**
 
-Question: How many issues were active during a certain period? 
+**Question:** How many issues were active during a certain period?
 
+## Overview
+This metric measures the number of issues that showed activity during a given period. Active issues are those that had comments, status changes, or were closed during the specified time frame. These issues indicate the level of interaction and work being done on a project, particularly in managing tasks, bugs, or feature requests.
 
-## Description
+Tracking active issues can help assess project activity and progress. High numbers of active issues may suggest an engaged community, while low numbers might indicate stagnation. For projects focused on DEI (Diversity, Equity, and Inclusion), this metric may help highlight participation patterns from different contributors.
 
-Issues are defined as in [Issues New](https://chaoss.community/metric-issues-new/).
-Issues showing some activity are those that had some comment,
-or some change in state (including closing the issue),
-during a certain period.
+## Want to Know More?
 
-For example, in GitHub Issues, a comment, a new tag, or
-the action of closing an issue, is considered as a sign of activity.
+<details>
```

Comment:
@yigakpoa hi!! Can you change this line to this:

`<span markdown="1"><details>`